### ISSUE TITLE
bug in spektral/examples/classification_delaunay.py?

### ISSUE THREAD

**Issue by wtwt5237** [danielegrattarola/spektral#7]
Hi Daniele

Thanks so much for creating spektral! A very useful tool for the community. I am running your spektral/examples/classification_delaunay.py example, and met with the following bug. Could you please take a look? Maybe I missed something? Everything before the "output..." line went ok.

Tao

```
>>> output = Dense(n_classes, activation='softmax')(pool)
Traceback (most recent call last):
  File "/home2/twang6/.conda/envs/spektral/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1659, in _create_c_op
    c_op = c_api.TF_FinishOperation(op_desc)
tensorflow.python.framework.errors_impl.InvalidArgumentError: Dimensions must be equal, but are 128 and 32 for 'dense_1/MatMul' (op: 'MatMul') with input shapes: [?,128], [32,2].

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home2/twang6/.conda/envs/spektral/lib/python3.6/site-packages/keras/engine/base_layer.py", line 457, in __call__
    output = self.call(inputs, **kwargs)
  File "/home2/twang6/.conda/envs/spektral/lib/python3.6/site-packages/keras/layers/core.py", line 879, in call
    output = K.dot(inputs, self.kernel)
  File "/home2/twang6/.conda/envs/spektral/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py", line 1085, in dot
    out = tf.matmul(x, y)
  File "/home2/twang6/.conda/envs/spektral/lib/python3.6/site-packages/tensorflow/python/ops/math_ops.py", line 2455, in matmul
    a, b, transpose_a=transpose_a, transpose_b=transpose_b, name=name)
  File "/home2/twang6/.conda/envs/spektral/lib/python3.6/site-packages/tensorflow/python/ops/gen_math_ops.py", line 5333, in mat_mul
    name=name)
  File "/home2/twang6/.conda/envs/spektral/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py", line 788, in _apply_op_helper
    op_def=op_def)
  File "/home2/twang6/.conda/envs/spektral/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py", line 507, in new_func
    return func(*args, **kwargs)
  File "/home2/twang6/.conda/envs/spektral/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 3300, in create_op
    op_def=op_def)
  File "/home2/twang6/.conda/envs/spektral/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1823, in __init__
    control_input_ops)
  File "/home2/twang6/.conda/envs/spektral/lib/python3.6/site-packages/tensorflow/python/framework/ops.py", line 1662, in _create_c_op
    raise ValueError(str(e))
ValueError: Dimensions must be equal, but are 128 and 32 for 'dense_1/MatMul' (op: 'MatMul') with input shapes: [?,128], [32,2].


```

**Comment by danielegrattarola** [comment]
Hi, 

this is due to a bug in `GlobalAttentionPool`, which was fixed in commit 548e393. Installing the latest version from source should solve the issue. I'll update the binary on PyPi soon. 

Cheers

**Comment by wtwt5237** [comment]
Thanks so much!

Tao
